src/Sampler.cpp.Rt

```diff
@@ -39,10 +39,9 @@ int Sampler::writeHistory(int curr_iter) {
 		tmp_loc.x = spoints[j].location.dx;
 		tmp_loc.y = spoints[j].location.dy;
 		tmp_loc.z = spoints[j].location.dz;
-     		fprintf(f,"Point coordinates: ");
-		csvWriteElement(f,tmp_loc);
 		for (int i = startIter; i< curr_iter; i++){
```

Comment:
@shkodm Please switch the places of the two loops - so that you'll have
for (iteration) { for (point) {write} } 
then it will be consistent throughout the file and easier to read.